lib/instrumentation/modules/redis.js

```diff
@@ -33,16 +35,24 @@ module.exports = function (redis, agent, { version, enabled }) {
   return redis
 
   function makeWrappedCallback (span, cb) {
-    return agent._instrumentation.bindFunction(function wrappedCallback () {
+    const wrappedCallback = agent._instrumentation.bindFunction(function () {
       if (span) span.end()
       if (cb) {
         return cb.apply(this, arguments)
       }
     })
+    wrappedCallback[isWrappedRedisCbSym] = true
+    return wrappedCallback
   }
 
   function wrapInternalSendCommand (original) {
     return function wrappedInternalSendCommand (commandObj) {
+      // Avoid re-wrapping internal_send_command called *again* in
+      // send_offline_queue for commands queued before "ready".
+      if (commandObj && commandObj.callback && commandObj.callback[isWrappedRedisCbSym]) {
```

Comment:
Question: I see we've altered the behavior of the wrapped `internal_send_command` but not the older wrapped `send_command`.  If offline queuing not a thing in the older version of the client with `send_command`, or is 2.5.3 too old to worry about? Or some third thing.